### PR TITLE
Ship the new ui

### PR DESCRIPTION
**Pull request by jakemcdermott**
##### SUMMARY
Serve `ui_next` as the default ui and remove the old ui code.

Screenshot from 2020-11-24 12-16-59

##### ADDITIONAL INFORMATION
There's some follow-up tasks related to this PR that we'll want to do after this merges:
- clean up old zuul configs: https://github.com/ansible/awx/issues/8649
- library license checks: https://github.com/ansible/awx/issues/8647
- upgrade page: https://github.com/ansible/awx/issues/8355
- clean up how we include build variables as part of src
- logo changes

### CODE REVIEW

**Review by rebeccahhh — APPROVED** [review]

**Comment by ryanpetrello** [comment]
recheck

**Comment by jakemcdermott** [comment]
recheck

File: awx/ui_next/src/api/models/Root.js [hunk 1]
```diff
@@ -25,6 +25,14 @@ class Root extends Base {
   logout() {
     return this.http.get(`${this.baseUrl}logout/`);
   }
+
+  readAssetVariables() {
+    // TODO: There's better ways of doing this. Build tools, scripts,
```
Comment (jakemcdermott) [inline awx/ui_next/src/api/models/Root.js:30]: I'll fix this elsewhere after merging. (See related [follow-up tasks](https://github.com/ansible/awx/pull/8607#issue-522632891) in the PR summary). 

In the meantime, this function works fine.

**Comment by jakemcdermott** [comment]
recheck

**Comment by unlikelyzero** [comment]
@jakemcdermott we will be performing a bug bash tomorrow AM. We'll use this build

**Review by wenottingham — APPROVED** [review]
```
+76 −184,854
```
APPROVED.

Seriously, this is amazing work by the whole team.

**Review by matburt — APPROVED** [review]
tenor

**Review by ryanpetrello — APPROVED** [review]
🚀🚀🚀🚀🚀🚀🚀

image

What an amazing accomplishment 🎉😮🎉!

**Comment by jakemcdermott** [comment]
cc: @mabashian @keithjgrant @marshmalien @nixocio @AlexSCorey @ and everyone else

Amazing work, everybody!

**Comment by jakemcdermott** [comment]
^^^^^^^^^^^^^^^^^^^^^^ @jlmitch5 @mabashian @keithjgrant @dsesami @kialam @marshmalien @nixocio @AlexSCorey @trahman73 @unlikelyzero @tiagodread and @ everyone 

You are spectacular. Amazing work, everybody!

**Review by AlanCoding — APPROVED** [review]

**Comment by dsesami** [comment]
nice job :)

**Comment by IPvSean** [comment]
woh woh woh, I LOVE IT
woh75

**Pull request merged**